### ISSUE TITLE
About pre-trained model of MSCeleb

### ISSUE THREAD

**Issue by RockPine520** [yaoing/DAN#13]
Thanks for your work which inspire me a lot! There are a few questions I would like to ask.

Could you tell me whether the MSCeleb-resnet18 pre-training model weights are trained by your own?
When I used the MSCeleb pre-training weights directly on resnet18 and trained the raf-db dataset according to your training strategy, I was able to achieve 88.75% accuracy, which I think is quite high. But in your paper, the accuracy of baseline resnet18 is just 86.25%, which confusing me a lot.

I‘m looking forward to your reply. Thank you!

**Comment by yaoing** [comment]
Thanks for your interest!
This `MSCeleb-resnet18` pretrained model is not trained by myself. In fact, it's also used in many other FER works like SCN and DACL. However, the best performances of all these model are below your stated 88.75%. 

If you're still unsure, try using their pre-trained `MSCeleb-resnet18` model for training and use it as a reference.

**Comment by RockPine520** [comment]
Thanks for your advice. I'll try it.
